src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/AzureSpatialAnchors/CustomSpatialAnchorManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace Microsoft.Azure.SpatialAnchors.Unity
+{
+    public static class CustomSpatialAnchorManagerExtensions
+    {
+        /// <summary>
+        /// Extension method that allows adding a SpatialAnchorManager with a custon SpatialAnchorConfig to a GameObject
```

Comment:
```suggestion
        /// Extension method that allows adding a SpatialAnchorManager with a custom SpatialAnchorConfig to a GameObject
```

Reply:
Also, should this instead read "allows adding a CustomSpatialAnchorManager" instead of "SpatialAnchorManager?"